be/src/exec/pipeline/pipeline_driver.cpp

```diff
@@ -264,6 +264,7 @@ StatusOr<DriverState> PipelineDriver::process(RuntimeState* runtime_state, int w
                     continue;
                 }
 
+                _try_to_release_buffer(runtime_state, curr_op);
```

Comment:
move it after L276?

Reply:
> move it after L276?

_try_to_release_buffer may change the behavior of `curr_op->has_output()` which is in the next line. I think we should keep it